src/components/Total/index.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import {IPlayer} from '../../types';
+import {getSum} from "../../utils";
+import {useState, useEffect} from "react";
+import styles from './Total.module.css';
+
+interface IProps {
+	players: Array<IPlayer>
+}
+
+export default function Total(props: IProps) {
+	const [winner, setWinner] = useState('null');
+
+	useEffect(() => {
+		setWinner(getWinner(props.players));
+	}, [props.players]);
+
+	function getWinner(players: Array<IPlayer>): string {
+		const bestPlayer =  players
```

Comment:
players.map(getScore).max()?

Reply:
I need to extract the `id` of the best player, not just his score